vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroup.java

```diff
@@ -675,6 +699,25 @@ protected boolean valueEquals(T value1, T value2) {
         return getItemId(value1).equals(getItemId(value2));
     }
 
+    /**
+     * If true, the user cannot interact with this element.
+     *
+     * @param disabled
+     *            the boolean value to set
+     */
+    protected void setDisabled(boolean disabled) {
+        getElement().setProperty("disabled", disabled);
+    }
+
+    /**
+     * If true, the user cannot interact with this element.
+     *
+     * @return the {@code disabled} property from the webcomponent
+     */
+    protected boolean isDisabledBoolean() {
+        return getElement().getProperty("disabled", false);
+    }
+
```

Comment:
- These could probably be marked private as we don't need to expose them for extensions.
- The name `isDisabledBoolean` could just be `isDisabled`

Reply:
Renamed and marked both as private.